{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.1",
	"cveMetadata": {
		"cveId": "CVE-2023-38734",
		"assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
		"assignerShortName": "ibm",
		"state": "PUBLISHED",
		"datePublished": "2023-08-22T21:18:08.392Z",
		"dateUpdated": "2024-08-02T17:46:56.808Z",
		"dateReserved": "2023-07-25T00:01:17.449Z"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "9a959283-ebb5-44b6-b705-dcc2bbced522",
				"shortName": "ibm",
				"dateUpdated": "2023-08-22T21:18:08.392Z"
			},
			"title": "IBM Robotic Process Automation privilege escalation",
			"descriptions": [
				{
					"lang": "en",
					"value": "\nIBM Robotic Process Automation 21.0.0 through 21.0.7.1 and 23.0.0 through 23.0.1 is vulnerable to incorrect privilege assignment when importing users from an LDAP directory.  IBM X-Force ID:  262481.\n\n",
					"supportingMedia": [
						{
							"type": "text/html",
							"base64": false,
							"value": "\n\n<span style=\"background-color: rgb(204, 217, 226);\">IBM Robotic Process Automation 21.0.0 through 21.0.7.1 and 23.0.0 through 23.0.1 is vulnerable to incorrect privilege assignment when importing users from an LDAP directory.  IBM X-Force ID:  262481.</span>\n\n"
						}
					]
				}
			],
			"affected": [
				{
					"vendor": "IBM",
					"product": "Robotic Process Automation",
					"defaultStatus": "unaffected",
					"versions": [
						{
							"status": "affected",
							"versionType": "semver",
							"version": "21.0.0",
							"lessThanOrEqual": "21.0.7.1"
						},
						{
							"status": "affected",
							"versionType": "semver",
							"version": "23.0.0",
							"lessThanOrEqual": "23.0.1"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"lang": "en",
							"description": "266 Incorrect Privilege Assignment"
						}
					]
				}
			],
			"metrics": [
				{
					"format": "CVSS",
					"scenarios": [
						{
							"lang": "en",
							"value": "GENERAL"
						}
					],
					"cvssV3_1": {
						"version": "3.1",
						"vectorString": "CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:H",
						"attackVector": "NETWORK",
						"attackComplexity": "HIGH",
						"privilegesRequired": "HIGH",
						"userInteraction": "NONE",
						"scope": "UNCHANGED",
						"confidentialityImpact": "HIGH",
						"integrityImpact": "HIGH",
						"availabilityImpact": "HIGH",
						"baseScore": 6.6,
						"baseSeverity": "MEDIUM"
					}
				}
			],
			"references": [
				{
					"tags": [
						"vendor-advisory"
					],
					"url": "https://www.ibm.com/support/pages/node/7028227"
				},
				{
					"tags": [
						"vdb-entry"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/262481"
				}
			],
			"source": {
				"discovery": "UNKNOWN"
			},
			"x_generator": {
				"engine": "Vulnogram 0.1.0-dev"
			}
		},
		"adp": [
			{
				"providerMetadata": {
					"orgID": "af854a3a-2127-422b-91ae-364da2661108",
					"shortName": "CVE",
					"dateUpdated": "2024-08-02T17:46:56.808Z"
				},
				"title": "CVE Program Container",
				"references": [
					{
						"tags": [
							"vendor-advisory",
							"x_transferred"
						],
						"url": "https://www.ibm.com/support/pages/node/7028227"
					},
					{
						"tags": [
							"vdb-entry",
							"x_transferred"
						],
						"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/262481"
					}
				]
			}
		]
	}
}
